docs/tools/cli/card.rst

```diff
@@ -0,0 +1,111 @@
+Command reference: ``avn card``
+==================================
+
+Here you’ll find the full list of commands for ``avn card``.
+
+
+Manage credit cards
+-------------------
+
+Commands for managing credit card details associated to Aiven projects.
+
+
+``avn card add``
+'''''''''''''''''''''''
+
+Adds a new credit card to the Aiven account.
+
+.. list-table::
+  :header-rows: 1
+  :align: left
+
+  * - Parameter
+    - Information
+  * - ``--cvc``
+    - Credit card security code
+  * - ``--exp-month``
+    - Card expiration month (1-12)
+  * - ``--exp-year``
+    - Card expiration year
+  * - ``--name``
+    - Name on the credit card
+  * - ``--number``
+    - Credit card number
+  * - ``--update-project``
+    - Assign card to a project
+
+**Example:** Add a credit card to a named project.
+
+::
+
+  avn card add --cvc 000         \
```

Comment:
I wonder if we should not use example values in the examples, such as name, card number. We can maybe agree on the standard values we can use for that?
A generic question - same goes for other examples here, and in other CLI commands.

Reply:
A good question, I updated some of the examples.